fabfile.py

```diff
@@ -342,10 +351,10 @@ def update(c, wiredtiger_branch=None, testy_branch=None):
         raise Exit("One or more errors occurred during update. Please retry the " \
                    f"update or run 'fab start' to restart {testy}.")
 
-# The workload function takes two optional arguments: upload and describe. If no arguments are
-# provided, the current workload is returned.
+# The workload function takes three optional arguments: upload, upload_config and describe. 
```

Comment:
I think this should be okay as upload - because it is uploading an entire workload directory and left for general purposes, but you can just specify if you want something specific?

Reply:
Let's leave it as is